### ISSUE TITLE
[Panel] Change svg default background in list views

### ISSUE THREAD

**Issue by jenstornell** [getkirby/kirby#665]
We had a discussion on Slack about the svg background.

**How Google does it:**

![image](https://user-images.githubusercontent.com/927011/42777593-ded57be0-893a-11e8-866d-167600e0af7d.png)

**Kirby 3 panel:**

![image](https://user-images.githubusercontent.com/927011/42777614-f03692d4-893a-11e8-87dd-7fa59898bc0e.png)

**What I did on Kirby Component Kit:**

https://github.com/jenstornell/kirby-component-kit/blob/master/assets/svg/chess-board.svg

(or in short)

```text
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8 8"><path fill="#fff" fill-rule="evenodd" d="M0 0h4v4H0V0zm4 4h4v4H4V4z"/></svg>
```

**In conclusion:** I think a bright background as default for svg files makes the most sense in lists like this. *Inside the image, in the file view, I think it's more ok to have a dark background.*

**Comment by wottpal** [comment]
Related: I was wondering if it actually makes sense to define the background in the `pages`-section and not in the file-template blueprint itself. So it must only be defined once.
